src/ess/sans/normalization.py

```diff
@@ -0,0 +1,63 @@
+import scipp as sc
+import scippneutron as scn
+
+
+def solid_angle(data, pixel_size, pixel_length):
+    """
+    Solid angle function taking pixel_size and pixel_lenght as parameters.
+    The method assumes pixels are rectangular.
+    """
+    L2 = scn.L2(data)
+    return (pixel_size * pixel_length) / (L2 * L2)
+
+
+def covert_and_rebin(data, wavelength_bins, begin, end):
```

Comment:
```suggestion
def convert_and_rebin(data, wavelength_bins, begin, end):
```
... but convert to what?